### ISSUE TITLE
About the train_set

### ISSUE THREAD

**Issue by xie0227** [braindecode/braindecode#354]
In tutorial, it seems that the shape of data in train_set `train_set = splitted['session_T']` is [channels, samples]，but when we use braindecode.models.eegnet.EEGNetv4 which hope input have [1,channels,samples]，it doesn't go wrong. I want to know why. Thank you very much.

**Comment by xie0227** [comment]
And I found the output of EEGNetv4 all  is negative and a little bit numerically large【-15~-10】. If I want to use the output as hidden feature, will it cause some troubles? I have used them to calculate SimCLR loss, but  I found loss is very large And can't converge, so I want to know is it because of this reason

**Comment by robintibor** [comment]
So the first dimension is the batch dimension so [1,channels,samples] would be batch size 1. Did you check the distribution of your input values? Maybe you can try normalizing/standardizing them to get smaller outputs as well.

**Comment by xie0227** [comment]
Thanks for your reply. I just omit batch size, and what I mean is  the data from dataloader which `dataset` is train_set `train_set = splitted['session_T'] ` is  [batch size, channels, samples] but for EEGNetv4, it hope input [batch size,1,channels, samples]. And when training, it doesn't go wrong, I want to know why.

**Comment by robintibor** [comment]
Maybe it had become clear to you already, but in 
https://github.com/braindecode/braindecode/blob/4ae629611675b3b8bd3d95818f327d5527ef7827/braindecode/models/eegnet.py#L79-L82
https://github.com/braindecode/braindecode/blob/4ae629611675b3b8bd3d95818f327d5527ef7827/braindecode/models/modules.py#L13-L17
https://github.com/braindecode/braindecode/blob/4ae629611675b3b8bd3d95818f327d5527ef7827/braindecode/models/eegnet.py#L187-L188
we ensure that the input has the correct 4d shape
